### ISSUE TITLE
'loosebidict' not available in bidict 0.14+

### ISSUE THREAD

**Issue by rodom** [derrix060/onedriveClient#9]
Having all sorts of issues getting this working. The latest:
```
rodom@oe2134:~$ onedrived start
Traceback (most recent call last):
  File "/usr/local/bin/onedrived", line 11, in <module>
    load_entry_point('onedrived==2.0.0', 'console_scripts', 'onedrived')()
  File "/usr/local/lib/python3.5/dist-packages/pkg_resources/__init__.py", line 572, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/local/lib/python3.5/dist-packages/pkg_resources/__init__.py", line 2752, in load_entry_point
    return ep.load()
  File "/usr/local/lib/python3.5/dist-packages/pkg_resources/__init__.py", line 2405, in load
    return self.resolve()
  File "/usr/local/lib/python3.5/dist-packages/pkg_resources/__init__.py", line 2411, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/local/lib/python3.5/dist-packages/onedrived/od_main.py", line 22, in <module>
    from .od_watcher import LocalRepositoryWatcher
  File "/usr/local/lib/python3.5/dist-packages/onedrived/od_watcher.py", line 6, in <module>
    from bidict import loosebidict
ImportError: cannot import name 'loosebidict'
```
Resolved by downgrading bidict to 0.13.1:
```sudo pip3 install --upgrade bidict==0.13.1```
However I guess, if this has been removed, the code needs updating to use something else.

**Comment by pintman** [comment]
Had the same issue. Thanks for the hint.

**Comment by derrix060** [comment]
Fix in c692b95c175576625bf816e38009d8bb480261d2
